app/main.py

```diff
@@ -2,0 +1,25 @@
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
Code Style: You should use type annotation for the `cafe` parameter. It is good practice to use type annotations as they make the code more readable and help catch potential errors. For example: `def go_to_cafe(friends: list, cafe: Cafe) -> str:`